### DISCUSSION TITLE
Expand Documentation

### DISCUSSION THREAD

**Discussion by jaydrogers** [serversideup/docker-php#167] [Ideas & Feature Requests]
# Problem
- The current documentation was a simple translation of what we currently had. There still seems to be a lot that should be explained to improve on-boarding

# Proposed Solution
Moved to the active issue tracking the solution: 
- https://github.com/serversideup/docker-php/issues/226

**Comment by alnouirah** [comment]
Yeah, that would be very nice especially for beginners or developers who know a little about Docker. 
1- Downloading the images ( done ).
2- Start development ( no ).

*Reason* 
does not know what to do next 😅.

**Comment by divdax** [comment]
Maybe some exmaple docker-compose.yml files for dev/production. I don't know how to setup my project. Can i use this image with Laravel Sail?

**Comment by jaydrogers** [comment]
Adding other comments from Discord:

https://discord.com/channels/910287105714954251/910287106427990038/1139927432909033482

> Maybe you can share in your documentations, a fully working docker-compose.yml too with laravel/symfony projects and mb with composer install. That will be helpful for another user.

**Comment by jaydrogers** [comment]
This discussion is being closed and locked. Everything has been moved to the active issue tracking the solution.

Thanks for your feedback everyone!

If you have any other comments, please add them here: https://github.com/serversideup/docker-php/issues/226
